package.json

```diff
@@ -6,9 +6,20 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "axios": "^0.19.2",
+    "history": "^4.10.1",
+    "lodash": "^4.17.15",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
-    "react-scripts": "3.4.0"
+    "react-redux": "^7.2.0",
```

Comment:
may I suggest the usage of [Redux Toolkit](https://redux-toolkit.js.org/)? as the project is still at the beginning it's still easy to set up and I guess it will be worth, it simplifies a lot the setup and you tend to write much less code